samples/notebooks/prediction/SDK_Custom_Predict_SDK_Integration.ipynb

```diff
@@ -0,0 +1,1531 @@
+{
```

Comment:
All these notebooks still need to move to the https://github.com/GoogleCloudPlatform/vertex-ai-samples repo. Work with the DPEs (Rosie is your initial POC) if you need help doing that.

Reply:
Yup, we've already moved them to the vertex-ai-samples repo. Chun-Hsiang, let's point the README to the new samples location, and delete these.

Reply:
We have added our notebooks to that repo. We will remove the notebooks from the main branch right after checking in this PR. Therefore we still have notebooks in the custom-prediction-routine branch.

Reply:
I'm not sure I'm following the logic here.

Why not remove these now? Why does it matter if they're in your dev branch?

Reply:
We are merging into main branch from custom-prediction-routine branch. We need to remove them from our dev branch in order to remove them from this PR.

Reply:
Or I remove them now and add them back after the merge. Let me do that now.

Reply:
Done.